Assets/MixedRealityToolkit.Providers/XRSDK/XRSDKBoundarySystem.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Boundary;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.XR;
+
+namespace Microsoft.MixedReality.Toolkit.XRSDK
+{
+    /// <summary>
+    /// The Boundary system controls the presentation and display of the users boundary in a scene.
+    /// </summary>
+    [HelpURL("https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Boundary/BoundarySystemGettingStarted.html")]
+    public class XRSDKBoundarySystem : BaseBoundarySystem
```

Comment:
Would it be better to support BoundarySystemProviders?

Reply:
I thought about this as well. We could, as I think we may run into platforms that support a boundary but don't support XR SDK or Unity's boundary APIs.
In this case, though, I think it's more similar to my other response. Devs will either be choosing XR SDK or Unity's legacy XR for their app, and won't need to (or can't, since Unity doesn't really) support both side-by-side.